### ISSUE TITLE
user prompted to set up a new auth method every login

### ISSUE THREAD

**Issue by SYoung3000** [catalyst/moodle-tool_mfa#350]
Hi all, we're seeing users that require auth prompted to set up a new authentication method on every login, despite the existing logins being listed on the login page. We're not seeing any setting that would control this. 

<img width="1038" alt="Screenshot 2022-05-19 at 13 48 57" src="https://user-images.githubusercontent.com/77289704/169299338-a8d8b51c-81dc-43f0-ad55-7d51f5ff9cbe.png">

**Comment by zozo77** [comment]
Same here.

**Comment by euneuber** [comment]
Same problem here: Admin completes "Authenticator App" logs out and on the next login no TOTP is requested but has to setup a new "Authenticator App".

There was a grace period of one week but this is over now. And it did work in this grace period...

**Comment by euneuber** [comment]
I turned the grace period off and now it uses the "Authenticator App" setup for the admin user!

Maybe the "grace period" plugin needs more/better documentation and help texts? Please explain the options of this plugin in "ELI5" mode!
